ert3/stats/_stats.py

```diff
@@ -15,11 +16,12 @@ def __init__(self, mean, std, *, size=None, index=None):
                 "Cannot create gaussian distribution with both size and index"
             )
 
+        self._index = index
         if size is not None:
             self._size = size
         else:
             self._size = len(tuple(index))
-        self._index = index
+            self._index = tuple(index)
```

Comment:
This seems a bit strange :thinking:

Reply:
I just wanted to avoid the situation where the index is an iterable that can only be consumed once. And I also wanted to make it (semi-)immutable.